common/app/model/content/Atoms.scala

```diff
@@ -86,7 +87,14 @@ object Atoms extends common.Logging {
     content.atoms.map { atoms =>
       val quizzes = extract(atoms.quizzes, atom => { Quiz.make(content.id, atom) })
 
-      val media = extract(atoms.media, atom => { MediaAtom.make(atom) })
+      val media = extract(atoms.media, atom => {
+        val endSlatePath = EndSlateComponents(
+          sectionId = content.sectionId.getOrElse(""),
+          shortUrl = content.fields.flatMap(_.shortUrl).getOrElse(""),
```

Comment:
worth confirming this behaviour with Editorial, though shouldn't block shipping this